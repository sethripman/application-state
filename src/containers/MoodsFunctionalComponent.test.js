import React from 'react';
import { shallow } from 'enzyme';
import MoodsFn from './MoodsFn';

describe('MoodsFn', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<MoodsFn />);
    expect(wrapper).toMatchSnapshot();
  });
});
