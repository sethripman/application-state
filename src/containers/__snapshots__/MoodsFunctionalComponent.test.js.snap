// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoodsFn matches a snapshot 1`] = `
<Fragment>
  <Controls
    dispatch={
      Array [
        Object {
          "count": 0,
          "func": [Function],
          "text": "Drink Coffee",
        },
        Object {
          "count": 0,
          "func": [Function],
          "text": "Snack",
        },
        Object {
          "count": 0,
          "func": [Function],
          "text": "Nap",
        },
        Object {
          "count": 0,
          "func": [Function],
          "text": "Study",
        },
        Object {
          "count": 0,
          "func": [Function],
          "text": "Exercise",
        },
      ]
    }
  />
  <Face
    emoji="ðŸ˜ "
  />
</Fragment>
`;
